recipes/cellprofiler/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = "CellProfiler" %}
+{% set version = "3.1.8" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+    url: "https://github.com/CellProfiler/{{ name }}/archive/v{{ version }}.tar.gz"
+    sha256: "6c327931c1a95b18d3f5615ba35c93df23bb5966448d7911e74664ed3ff7e230"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install --no-deps --ignore-installed . -vv"
+  skip: True # [win]
+  skip: True # [osx]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python <3
+    - pip
+    - cython
+    - openjdk 8.*
+    
+  run:
+    - python <3
+    - boto3
+      #    - centrosome==1.1.6
+      #    - docutils==0.14
+    - h5py
+      #    - inflect==2.1.0
+    - wxpython
+    - joblib
+    - mahotas
+    - matplotlib==2.2.3
```

Comment:
I don't think this strict pin is necessary.

Reply:
yes, it needs to be > 2.0.0, so I will change it to >=2.0